shared/src/main/java/uk/gov/di/authentication/shared/entity/ClientSession.java

```diff
@@ -70,4 +93,16 @@ public ClientSession setDocAppSubjectId(Subject docAppSubjectId) {
     public String getClientName() {
         return clientName;
     }
+
+    private List<VectorOfTrust> orderVtrList() {
+        return this.vtrList.stream()
+                .sorted(
+                        Comparator.comparing(
+                                        VectorOfTrust::getLevelOfConfidence,
+                                        Comparator.nullsFirst(Comparator.naturalOrder()))
+                                .thenComparing(
+                                        VectorOfTrust::getCredentialTrustLevel,
+                                        Comparator.nullsFirst(Comparator.naturalOrder())))
+                .collect(Collectors.toList());
+    }
```

Comment:
What's the difference between this and `VectorOfTrust.orderVtrList`?

Reply:
Good spot, shall change 👍